libraries/fiff/fiff_coord_trans.cpp

```diff
@@ -162,6 +162,35 @@ bool FiffCoordTrans::read(QIODevice& p_IODevice, FiffCoordTrans& p_Trans)
 
 //=============================================================================================================
 
+void FiffCoordTrans::write(QIODevice &p_IODevice)
+{
+    //
+    //   Open the file, create directory
+    //
+
+    // Create the file and save the essentials
+    FiffStream::SPtr pStream = FiffStream::start_file(p_IODevice);
+    printf("Write coordinate transform in %s...\n", pStream->streamName().toUtf8().constData());
+    this->writeToStream(pStream.data());
+    pStream->close();
+}
+
+//=============================================================================================================
+
+void FiffCoordTrans::writeToStream(FiffStream* pStream)
+{
+    pStream->start_block(FIFFB_MEAS);
+    pStream->start_block(FIFFB_MEAS_INFO);
+
+    pStream->write_coord_trans(*this);
+
+    pStream->end_block(FIFFB_MEAS_INFO);
+    pStream->end_block(FIFFB_MEAS);
+    pStream->end_file();
+}
```

Comment:
did you test the read/write values with a read-write-read example? Please create a `test_fiff_coord_trans` for this.

Reply:
I tested it briefly within the fiff_rwr, but did not included it into it.